### PR TITLE
NAS-104754 / 12.0 / Correct logo/brand for TN 12

### PR DESCRIPTION
**Pull request by sonicaj**

### CODE REVIEW

File: src/freenas/boot/lua/logo-TrueNAS.lua [hunk 1]
```diff
@@ -43,7 +43,54 @@ local truenas_color = {
...
+"      /\\_________!|!__!|//",
+"      /?___________?|!__!\\/",
+"     //!____________?/|!__?//",
+"    //?______________|/\\!__!\\/",
+"   //?_______________?///|!_!\\/",
+"  /\\?__________!!!!!?||||||!__|//",
+"///|!!!!!!!!!!!!!!!!!!!!!!!!!!?///",
+}
+
+local odl2_truenas_color = {
```
Comment (freqlabs) [inline src/freenas/boot/lua/logo-TrueNAS.lua]:
```suggestion
local old2_truenas_color = {
```

**Comment by sonicaj** [comment]
True, history is a mess on this one :p

File: src/freenas/boot/loader.conf [hunk 2]
```diff
@@ -1,11 +1,11 @@
 #
 # Boot loader file for FreeNAS.
```
Comment (freqlabs) [inline src/freenas/boot/loader.conf]: Whoops missed a FreeNAS ^

Comment (sonicaj) [reply src/freenas/boot/loader.conf]: Whoops, will correct it ;)

**Pull request merged**